build.ps1

```diff
@@ -210,24 +220,33 @@ if (!$SkipBuild)
             Remove-Item -Force -Recurse "$repoPath\obj\"
         }
 
-        # NuGet restore disregards platform/configuration
         # dotnet restore defaults to Debug|Any CPU, which requires the /p:platform specification in order to replicate NuGet restore behavior.
-        $restoreArgsInit = "restore $repoPath\InstrumentationEngine.sln --configfile $repoPath\NuGet.config"
-        $restoreArgs = @(
-            "$restoreArgsInit /p:platform=`"x86`""
-            "$restoreArgsInit /p:platform=`"x64`""
-            "$restoreArgsInit /p:platform=`"Any CPU`""
-        )
-        Invoke-ExpressionHelper -Executable "dotnet" -Arguments $restoreArgs -Activity 'dotnet Restore Solutions'
+        $dotnetRestoreArgs = "restore `"$repoPath\InstrumentationEngine.sln`" --configfile `"$repoPath\NuGet.config`" /p:platform=`"Any CPU`""
+        if ($ARM64)
+        {
+            $dotnetRestoreArgs = "$dotnetRestoreArgs /p:IncludeARM64='True'"
+        }
+
+        Invoke-ExpressionHelper -Executable "dotnet" -Arguments $dotnetRestoreArgs -Activity 'dotnet Restore Solutions'
+
+        # NuGet restore disregards platform/configuration
+        $nugetRestoreArgs = "restore `"$repoPath\NativeNugetRestore.sln`" -configfile `"$repoPath\NuGet.config`""
+        Invoke-ExpressionHelper -Executable "nuget" -Arguments $nugetRestoreArgs -Activity 'nuget Restore Solutions'
```

Comment:
Why are you using `nuget` instead of `dotnet`? Doesn't the restoration for the `x64` and `x86` platforms still need to take place?

Reply:
I've added nuget to mirror our build pipeline, but seems I left off x86/x64 restore. I'll get those readded.

Reply:
Actually, the dotnet restore is only done for AnyCPU so we don't need x86/x64 differentiation.